Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,39 @@
-function setAlarm() {}
+function setAlarm() {
+  // Accessing and storing input and display elements from html file
+  let timeDisplay = document.getElementById("timeRemaining");
+  let inputField = document.getElementById("alarmSet");
+
+  // Converting input value to a number
+  let seconds = parseInt(inputField.value);
+
+  // Checking if input is valid (not empty, not negative, and a number)
+  if (isNaN(seconds) || seconds <= 0) {
```

Comment:
I see you're doing a good job checking that the input is a number and it's greater than 0. 
Would it be a good idea to check if the user enters a decimal?